src/Orc.FilterBuilder.Xaml/ViewModels/EditFilterViewModel.cs

```diff
@@ -89,15 +82,18 @@ public override string Title
         public bool EnableAutoCompletion { get; private set; }
         public bool AllowLivePreview { get; private set; }
         public bool EnableLivePreview { get; set; }
-
+        public bool IsLivePreviewDirty { get; set; }
```

Comment:
public setter really required?

Reply:
should be removed...also made other refactoring in property accessors